<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarCompareWindow</class>
 <widget class="QDialog" name="CarCompareWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сравнение автомобилей</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 240, 239);
color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: none;
border-width: 0px;</string>
  </property>
  <widget class="QLabel" name="toLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;</string>
   </property>
   <property name="text">
    <string>Сравнение 2 автомобилей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 9pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Год выпуска:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mileageLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Пробег(км):</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Стоимость $:</string>
   </property>
  </widget>
  <widget class="QLabel" name="brandLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Марка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Модель:</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Емкость
аккумулятора:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fuelLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Объем топливного
бака:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hybridTypeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Тип гибрида:</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarYearLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarMileageLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarPriceLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarBrandLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarModelLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarBatteryLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarFuelLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarHybridTypeLabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarYearLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarBrandLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarMileageLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarModelLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarHybridTypeLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarBatteryLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarPriceLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarFuelLabel2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: #E0D5D2;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
